addons/csw/stringtable.xml

```diff
@@ -572,6 +620,7 @@
         </Key>
         <Key ID="STR_ACE_CSW_m2_gun">
             <English>[CSW] M2 Gun Bag</English>
```

Comment:
@TheCandianVendingMachine I'd include some more info about the weapon here, above and beyond the model number. HMG? Browning? Something along those lines.

Reply:
`M2 Browning` would be my choice for addendum